instrumentation/github.com/Shopify/sarama/otelsarama/go.mod

```diff
@@ -8,5 +8,6 @@ require (
 	github.com/Shopify/sarama v1.32.0
 	github.com/stretchr/testify v1.7.1
 	go.opentelemetry.io/otel v1.7.0
+	go.opentelemetry.io/otel/sdk v1.7.0
```

Comment:
the tests using the SDK are in `instrumentation/github.com/Shopify/sarama/otelsarama/test`

Reply:
I can't move it into the test subpackage, as the `config` struct is private.
I've taken another approach, setting up a fake tracerprovider/tracer to avoid having the SDK.